homeassistant/components/google/icons.json

```
@@ -0,0 +1,6 @@
+{
+  "services": {
+    "add_event": "mdi:calendar",
+    "create_event": "mdi:calendar"
```

```suggestion
    "add_event": "mdi:calendar-plus",
    "create_event": "mdi:calendar-plus"
```
The [base calendar integration](https://github.com/home-assistant/core/blob/2b19783311cba3e9f50deb91bf735438dc6266fc/homeassistant/components/calendar/icons.json#L12) is already using `mdi:calendar-plus`